### DISCUSSION TITLE
Can i restore specific files from pv snapshot ?

### DISCUSSION THREAD

**Discussion by ahmedTouati** [vmware-tanzu/velero#3930] [Community support Q&A]
I'm doing daily snapshots on pv in my cluster using velero i wonder if I can only restore specific file from snapshot, or should i restore the entire snapshot to a new volume than select manually the needed files and copy them to the old volume ?

**Comment by zubron** [comment]
Velero does not support specific file backup, our unit of backup is a PV, so you will need to restore the snapshot and extract the files needed.

**Reply by ahmedTouati** [reply]
Ok, Thanks
